pkgs/applications/misc/omorfi/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, autoreconfHook
+, cg3
+, fetchFromGitHub
+, hfst
+, hfst-ospell
+, icu
+, libvoikko
+, pkg-config
+, python3
+, python3Packages
+, zip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "omorfi";
+  version = "0.9.9";
+
+  src = fetchFromGitHub {
+    owner = "flammie";
+    repo = "omorfi";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-UoqdwNWCNOPX6u1YBlnXUcB/fmcvcy/HXbYciVrMBOY=";
+  };
+
+  # Fix for omorfi-hyphenate.sh file not found error
+  postInstall = ''
+    mv $out/share/omorfi/omorfi.hyphenate-rules.hfst $out/share/omorfi/omorfi.hyphenate.hfst
+  '';
+
+  nativeBuildInputs = [
+    autoreconfHook
+    cg3
+    pkg-config
+    python3
+    libvoikko
+    zip
+  ];
+
+  buildInputs = [
+    hfst-ospell
+  ];
+
+  propagatedBuildInputs = [
+    hfst
+    python3Packages.hfst-python
+    icu
+  ];
```

Comment:
Do we really need to propagate all those? How are they used?

Reply:
Headers from  `hfst` and `icu` are included in `omorfi`'s C++ headers.

`hfst-python` is required to run some of the programs in `$out/bin` (at least `omorfi-vislcg.bash`)

Reply:
> `hfst-python` is required to run some of the programs in `$out/bin` (at least `omorfi-vislcg.bash`)

Then those programs should rather be wrapped or patched to find those dependencies.



> Headers from `hfst` and `icu` are included in `omorfi`'s C++ headers.

Is there no .pc or cmake file which references those headers?

Reply:
Looks like `hfst` is referenced in omorfi.pc (though it's referring to an incorrect include path), but `icu` isn't referred at all in either `omorfi` or `hfst` pkg-config files.